sdk/servicebus/service-bus/test/atomManagement.spec.ts

```diff
@@ -917,7 +1109,9 @@ const alwaysBeExistingRule = "alwaysbeexistingrule";
         "_response",
         "createdOn",
         "updatedOn",
-        "accessedOn"
+        "accessedOn",
+        "forwardTo",
+        "forwardDeadLetteredMessagesTo"
```

Comment:
Why would the response for a create rule operation have the forward related attributes?